csrc/python_frontend/fusion_record.h

```diff
@@ -726,48 +726,40 @@ struct SqueezeOpRecord : RecordFunctor {
 };
 
 //! Specialized Record Functor for the FusionState's broadcast_in_dim op.
-
-template <typename OutputShapeType>
+// NOTE: output_size gives the rank of the output tensor.  This size can be
```

Comment:
Could call it `output_dims` or something instead? `output_size` seems to suggest it's a length of something, it just wasn't immediately clear to me it's the length of the shape of the output.

Reply:
This was a good point.  I changed the name to `output_ndims`.